src/components/SnippetBlock/index.js

```diff
@@ -4,60 +4,30 @@ import CodeBlock from '@theme/CodeBlock'
 const SnippetBlock = ({
   children,
   funcName = 'main',
-  funcEnd = '}',
-  snippets,
   leadingSpaces = 2,
+  dropHead = 0,
```

Comment:
Can you add a comment on what these two new parameters mean?

Reply:
will do, they let you drop leading or trailing lines (e.g., if you want to not show the return statement).